src/d3d9/d3d9_swapchain.cpp

```diff
@@ -677,6 +697,11 @@ namespace dxvk {
     if (unlikely(pRamp == nullptr))
       return;
 
+    if (unlikely(!validateGammaRamp(pRamp->red)
+              || !validateGammaRamp(pRamp->blue)
+              || !validateGammaRamp(pRamp->green)))
+      return;
```

Comment:
Is this correct? If I read the wine tests correctly, only one channel needs to satisfy the conditions.

Reply:
I changed it to only reject if all channels are not valid since it's enough for this stupid game.